SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/entity/ShippingCost.java

```diff
@@ -46,7 +56,7 @@ public void setType(String type) {
 		this.type = type;
 	}
 
```

Comment:
@KendraMBach Lolz. Does the customer ever see this as separate from their order total?